Documentation/Diagnostics/PH2139.md

```diff
@@ -0,0 +1,51 @@
+# PH2139: Enable documentation creation
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2139 |
+| Category  | [Documentation](../Documentation.md) |
+| Analyzer | [EnableDocumentationCreationAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/EnableDocumentationCreationAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
```

Comment:
I've been adding a row for "Enabled By Default"

Reply:
Added here also